src/window/window_wrapper/mod.rs

```diff
@@ -252,8 +252,8 @@ impl GlutinWindowWrapper {
         }
 
         let horizontal_input_type = match y {
```

Comment:
just noticed, I think this y should be x. Actually its a bit odd to have a match where the actual value isn't used.

Reply:
```suggestion
        let horizontal_input_type = match x {
```